src/niweb/apps/noclook/views/detail.py

```diff
@@ -661,3 +663,43 @@ def group_detail(request, handle_id):
     location_path = node.get_location_path()
     return render(request, 'noclook/detail/group_detail.html',
                   {'node_handle': nh, 'node': node, 'location_path': location_path})
+
+@login_required
+def role_detail(request, role_name):
```

Comment:
Copy paste error? role_detail function is defined twice.

Reply:
My fault, I've removed all previous references to Role as it was, but forgot to remove that last one. It's done now.